src/gpu/pannotia/pagerank/pagerank.cpp

```diff
@@ -198,12 +345,12 @@ int main(int argc, char **argv)
         // Launch pagerank kernel 1
         hipLaunchKernelGGL(HIP_KERNEL_NAME(pagerank1), dim3(grid), dim3(threads), 0, 0, row_d, col_d, data_d, pagerank1_d,
                                       pagerank2_d, num_nodes, num_edges);
-
+        hipDeviceSynchronize();
```

Comment:
I guess you added these here because this is to a different repo than your other commit?

Reply:
Fine to leave these in here for our internal repo, but for other benchmarks we should push our hipDeviceSynchronize changes as a separate commit.